orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -22,13 +22,13 @@ namespace api {
 
 namespace {
 
-constexpr const char* k_tensor_proto_file_name = "tensors.pbseq";
-constexpr const char* k_tensor_proto_properties_file_name = "properties.pbseq";
-constexpr const char* k_trainable_param_root_prefix = "paramtrain";
-constexpr const char* k_non_trainable_param_root_prefix = "paramfrozen";
-constexpr const char* k_optimizer_root_prefix = "optim";
-constexpr const char* k_property_root_prefix = "custom";
-constexpr const char* k_name_seperator = "_";
+const PathString k_tensor_proto_file_name = ORT_TSTR("tensors.pbseq");
```

Comment:
how about using `constexpr auto k_tensor_proto_file_name = ORT_TSTR("tensors.pbseq");`

Reply:
changing to const PatchChar* will introduce string copying unless I modify some function signatures.

Reply:
The benefit is doing things at compile time as far as possible. Another idea is to use `std::string_view`.
I guess, if this will require a lot of changes on your end, we can skip it.